### PR TITLE
Add some logger info while reading data

### PR DESCRIPTION
**Pull request by pranayaryal**
### Code contributions
Major - and most appreciated - contribution is pull request with feature or bug fix. Each pull request initiates discussion about your code contribution.

Each pull request should be provided with minimal description about its contents.
#

Thanks!

Jakub & Kamil,

_core contributors to the minerva.ml_

### CODE REVIEW

**Comment by Ninoko** [comment]
Why you're removing `.yaml` files from config?

File: configs/neptune.yaml [hunk 1]
```diff
@@ -1,135 +0,0 @@
-project: ORGANIZATION/home-credit
```
Comment (jakubczakon) [inline configs/neptune.yaml]: @pranayaryal We do need those configs to parse parameters.

Comment (pranayaryal) [reply configs/neptune.yaml]: What's the best way to gitignore personal configs?

Comment (jakubczakon) [reply configs/neptune.yaml]: Well we usually just create neptune_local.yaml and then add that to .gitignore

**Comment by kamil-kaczmarek** [comment]
Hi @pranayaryal,

First of all thank you for your contribution! I can see that you also opened newer version of this PR. If you don't mind I'll close this one and proceed with the newer one.

Thanks
Kamil

**Pull request closed without merging**